src/formatDate.js

```diff
@@ -8,7 +8,75 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
-}
+  // create property for symbols from and to
+  const simbolFormat = fromFormat[fromFormat.length - 1];
+  const simbolTo = toFormat[toFormat.length - 1];
+  // create split array from date
+  const dateDay = date.split(simbolFormat);
+  // create property that will save new date
+  let year = '';
+  let day = '';
+  let month = '';
+
+  // add value for year, day, month from dateDay
+  for (let i = 0; i < fromFormat.length - 1; i++) {
+    switch (fromFormat[i]) {
+      case 'YYYY':
+        year = dateDay[i];
+        break;
+      case 'YY':
+        year = dateDay[i];
+        break;
+      case 'MM':
+        month = dateDay[i];
+        break;
+      case 'DD':
+        day = dateDay[i];
+        break;
+      default:
+        break;
+    }
+  }
```

Comment:
The 'switch' statement inside this 'for' loop can be simplified. Since both 'YYYY' and 'YY' cases perform the same action, you could merge them into a single case with 'case 'YYYY': case 'YY':'.